pkg/api/v1/schema.go

```diff
@@ -35,8 +35,15 @@ type DomainSpec struct {
 }
 
 type Memory struct {
-	Value uint   `json:"value"`
-	Unit  string `json:"unit"`
+	// Request field defines the amount of memory a VM gets upon starting
+	// The format follows k8s quantity specification
+	Request string `json:"request"`
+
+	// Limit defines the maximum amount of memory a VM can ever get in kiloBytes.
+	// This field depends on qemu capabilities, but can be only 4x as big as
+	// `Request` at the time I am writing this (8/2017)
+	// The format follows k8s quantity specification
+	Limit string `json:"limit"`
```

Comment:
Could you use "limits" here, to allign with Pods semantics?

Reply:
Well the plural form does not make sense in this case. It might if we decide to use limits.memory instead of memory.limit.